File: Data/Interfaces/IProject.cs
namespace Data.Interfaces;

public interface IProject
{
    public string Name { get; set; }
    public DateTime StartTime { get; set; }
    public DateTime EndTime { get; set; }
    public int ProjectManager { get; set; }
    public int Customer { get; set; }
    public string Service { get; set; }
    public decimal TotalPrice { get; set; }
    public string Status { get; set; }
}

File: frontend/src/interfaces/IProject.ts
import { ICustomer } from "./ICustomer";
import { IService } from "./IService";
import { IUser } from "./IUser";

export interface IProject {
    id: number;
    name: string;
    startTime: string;
    endTime: string;
    status: string;
    totalPrice: number;
    customer: ICustomer;
    service: IService;
    projectManager: IUser;
}